tubesync/sync/models.py

```diff
@@ -287,6 +287,13 @@ class IndexSchedule(models.IntegerChoices):
         help_text=_('If "delete old media" is ticked, the number of days after which '
                     'to automatically delete media')
     )
+    filter_text = models.CharField(
+        _('filter string'),
+        max_length=100,
+        default='.*',
```

Comment:
This should probably default to blank and not run a regex at all unless specifically set, otherwise this will be running a `.*` pointless regex against every item.

Reply:
I struggled a bit with default blank behaviour and this was my fix, but I can see why this is undesirable. I'll put an "if filter_text:" to suppress wherever the regex is called for the blank behaviour and double check anywhere this doesn't work.